apps/daimo-mobile/src/model/account.ts

```diff
@@ -97,6 +97,10 @@ export type Account = {
 
   /** True once we've completed onboarding. */
   isOnboarded: boolean;
+
+  /** Request data for notifications */
+  requests: DaimoRequestV2Info[];
```

Comment:
Looks like the old `trackedRequests` is unused. We can delete it now that we have `requests`.